sigh/bin/sigh

```diff
@@ -67,6 +68,7 @@ class SighApplication
                'The path may be prefixed with a identifier in order to determine which provisioning profile should be used on which app.',
                &multiple_values_option_proc(c, "provisioning_profile", &proc { |value| value.split('=', 2) })
       c.option '-d', '--display_name STRING', String, 'Display name to use'
+      c.option '-g', '--new_bundle_id STRING', String, 'New application bundle ID'
```

Comment:
It should be noted here that only the main app bundle ID will be changed. The `-b` flag of `resign.sh` does not apply for nested apps and extensions.